osmosis-installer.py

```diff
@@ -208,10 +208,10 @@ def stateSyncInit ():
     #TRUST_HEIGHT= str(int(LATEST_HEIGHT.stdout.strip()) - 2000)

```

Comment:
Do we need to keep this code?

Reply:
Yes, since statesync will be back up soon. I can remove it for cleanliness but thought I might as well just comment it out for now until we get statesync back